inference-engine/src/transformations/include/transformations/low_precision/disable_convert_constant_folding_on_const_path.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <utility>
+
+#include <transformations_visibility.hpp>
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API DisableConvertConcstantFoldingOnConstPath;
```

Comment:
Why you put this transformation to transformations_library?

Reply:
moved to LPT, thanks